django_spanner/operations.py

```diff
@@ -133,13 +251,32 @@ def get_db_converters(self, expression):
             converters.append(self.convert_uuidfield_value)
         return converters
 
+    # TODO: here and below, remove unused expression and connection parameters
+    # or include them to the code.
     def convert_binaryfield_value(self, value, expression, connection):
+        """Convert a binary field to Cloud Spanner.
```

Comment:
AFAICT these docstrings are backwards: `value` comes from python-spanner, and we're converting from one `bytes` (or ["bytes-like object"](https://docs.python.org/3/glossary.html#term-bytes-like-object)) it into another: from a base64-encoded `bytes` returned by spanner to a non-base64-encoded `bytes` for django, since that's the internal type for django `BinaryField`s.